sdk/tables/azure-data-tables/tests/test_table_batch.py

```diff
@@ -249,10 +249,10 @@ def test_batch_update(self, tables_storage_account_name, tables_primary_storage_
             entity = TableEntity()
             entity.PartitionKey = u'001'
             entity.RowKey = u'batch_update'
-            entity.test = EntityProperty(True)
+            entity.test = EntityProperty(True, EdmType.BOOLEAN)
             entity.test2 = u'value'
             entity.test3 = 3
-            entity.test4 = EntityProperty(1234567890)
+            entity.test4 = EntityProperty(1234567890, EdmType.INT32)
```

Comment:
Would be good to either add a test, or modify one of the existing ones to just use naked tuples:
`entity.test4 = (1234567890, 'Edm.Int32')`